tgui/packages/tgui/interfaces/common/PressureBar.js

```diff
@@ -0,0 +1,54 @@
+import { Box, ProgressBar, AnimatedNumber } from '../../components';
+
+
+
```

Comment:
Surprised the linter doesn't yell for this many blank lines?

Reply:
It doesn't, no. Should definitely be added to the linter though.